_cxx/envoy.mk

```diff
@@ -18,8 +25,15 @@ ENVOY_TEST_LABEL ?= //test/...
   ENVOY_FULL_DOCKER_TAG ?= $(ENVOY_DOCKER_REPO):envoy-full-$(ENVOY_DOCKER_VERSION)
 # END LIST OF VARIABLES REQUIRING `make update-base`.
 
-# for builder.mk...
-export ENVOY_DOCKER_TAG
+# How to set ENVOY_GO_CONTROL_PLANE_COMMIT: In github.com/envoyproxy/go-control-plane.git, the majority
+# of commits have a commit message of the form "Mirrored from envoyproxy/envoy @ ${envoy.git_commit}".
+# Look for the most recent one that names a commit that is an ancestor of our ENVOY_COMMIT.  If there
+# are commits not of that form immediately following that commit, you can take them in too (but that's
+# pretty uncommon).  Since that's a simple sentence, but it can be tedious to go through and check
+# which commits are ancestors, I added `make guess-envoy-go-control-plane-commit` to do that in an
+# automated way!  Still look at the commit yourself to make sure it seems sane; blindly trusting
+# machines is bad, mmkay?
+ENVOY_GO_CONTROL_PLANE_COMMIT = f1f47757da33f7507078cf7e9e60915418c7bd10
```

Comment:
Why tagging a SHA vs a release?

Reply:
Because it's what `make guess-envoy-go-control-plane-commit` says best matches our version of Envoy.